internal/controllers/tenant/tenant_reconciler_function_test.go

```
@@ -482,18 +482,18 @@ var _ = Describe("Deletion", func() {
+		tenant := privatev1.Tenant_builder{
 			Id: "org-123",
 			Metadata: privatev1.Metadata_builder{
 				Name:              "test-org",
 				Finalizers:        []string{finalizers.Controller},
 				DeletionTimestamp: deletionTimestamp,
 			}.Build(),
-			Status: privatev1.OrganizationStatus_builder{
-				State:               privatev1.OrganizationState_ORGANIZATION_STATE_SYNCED,
-				IdpOrganizationName: "test-org",
+			Status: privatev1.TenantStatus_builder{
+				State:               privatev1.TenantState_TENANT_STATE_SYNCED,
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)